src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -211,11 +205,12 @@ def load_command_table(self, _):
     from .operations.dns import DNSListReferences
     self.command_table["network dns list-references"] = DNSListReferences(loader=self)
 
-    with self.command_group('network dns zone', network_dns_zone_sdk) as g:
+    with self.command_group('network dns zone') as g:
+        from .custom import DNSZoneUpdate
```

Comment:
This is not needed if you register the command. It will show up on its own.

Reply:
Fixed, Removed